test/ValidateID.cpp

```diff
@@ -406,7 +406,7 @@ class OpTypeArrayLengthTest
     : public spvtest::TextToBinaryTestBase<::testing::TestWithParam<int>> {
  protected:
   OpTypeArrayLengthTest()
-      : position_{0, 0, 0}, diagnostic_(spvDiagnosticCreate(&position_, "")) {}
+      : position_(), diagnostic_(spvDiagnosticCreate(&position_, "")) {}
```

Comment:
Ew. You're right!  It's a POD type, and so leaves values in an indeterminate state.
So good catch!    Best to explicitly set the values in the  actual code of this constructor, then?